examples/00_quick_start/sasrec-tf2.ipynb

```diff
@@ -0,0 +1,321 @@
+{
```

Comment:
<p>In other notebooks we have all the parameters in the same cell, the reasons are first because we might want to inject parameters programmatically with papermill (see https://github.com/microsoft/recommenders/tree/main/tests#how-to-create-tests-on-notebooks-with-papermill) and also for simplicity</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1530/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>